cosmos/aggevm.json

```diff
@@ -0,0 +1,45 @@
+{
+  "rpc": "https://aggevm-rpc.web3-idea.xyz",
+  "rest": "https://aggevm-api.web3-idea.xyz",
+  "nodeProvider": {
+    "name": "Web3 Idea",
+    "email": "zhangzhishun1024@gmail.com",
+    "website":"https://web3-idea.xyz"
+  },
+  "chainId": "aggevm",
+  "chainName": "Agg",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/aggevm/chain.png",
+  "stakeCurrency": {
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/aggevm/chain.png"
+  },
+  "bip44": {
+    "coinType": 118
+  },
+  "bech32Config": {
+    "bech32PrefixAccAddr": "agg",
+    "bech32PrefixAccPub": "aggpub",
+    "bech32PrefixValAddr": "aggvaloper",
+    "bech32PrefixValPub": "aggvaloperpub",
+    "bech32PrefixConsAddr": "aggvalcons",
+    "bech32PrefixConsPub": "aggvalconspub"
+  },
+  "currencies": [
+    {
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/aggevm/chain.png"
```

Comment:
@zhangzhishun Could you add `currencies ` information?
and `coinImageUrl` should be `uagg.png` better